lib/src/internal/control_client.dart

```diff
@@ -28,15 +33,33 @@ class ControlClient implements AbstractControlClient {
         }, timeout: _configuration.transportStrategy.grpcConfig.deadline));
   }
 
+  Future<Map<String, String>> makeHeaders({String? cacheName}) async {
+    var headers = <String, String>{};
+    if (cacheName != null) {
+      headers.addEntries({'cache': cacheName}.entries);
+    }
+    if (firstRequest) {
+      firstRequest = false;
+      try {
+        String? packageVersion = await findPackageVersion();
+        headers.addEntries({'agent': packageVersion ?? 'unknown'}.entries);
```

Comment:
if i'm reading this right, the header would only include the version number; is that accurate? we want to make sure it includes the word "dart". even if the version number is unknown.

but lmk if i'm reading the code wrong!

Reply:
oh yeah, you're right, I can add "dart" in there